src/python/doc/wasserstein_distance_user.rst

```diff
@@ -9,12 +9,21 @@ Definition
 
 .. include:: wasserstein_distance_sum.inc
 
-This implementation is based on ideas from "Large Scale Computation of Means and Cluster for Persistence Diagrams via Optimal Transport".
+Functions
+---------
+This implementation uses the Python Optimal Transport library and is based on
+ideas from "Large Scale Computation of Means and Cluster for Persistence
+Diagrams via Optimal Transport".
 
-Function
---------
 .. autofunction:: gudhi.wasserstein.wasserstein_distance
 
+This other implementation comes from `Hera
+<https://bitbucket.org/grey_narn/hera/src/master/>`_ (BSD-3-Clause) and is
+based on `"Geometry Helps to Compare Persistence Diagrams."
+<http://dx.doi.org/10.1137/1.9781611974317.9>`_ by Michael Kerber, Dmitriy
+Morozov, and Arnur Nigmetov, at ALENEX 2016.
```

Comment:
Why didn't you use the `:cite:` tag and a new reference in the *biblio/bibliography.bib* ?
I agree that the rendering is not really satisfying...

Reply:
Because some people read the doc by calling `help(gudhi.hera.wasserstein_distance)` instead of reading the sphinx-generated webpage. But I think it is ok if those people don't get the link, I'll use bibtex.